### ISSUE TITLE
the problem when using the cellpose 4

### ISSUE THREAD

**Issue by yuu342** [MouseLand/cellpose#1430]
Hellp,I met a problem when I try to use the cellpose_SAM .

My image has shape: (1969, 2000, 3). Assuming channel dimension is last with 3 channels.And the following is my code.However,I found that it seemed to have divided the background without dividing the tissue.Could you please help me correct my codes?
first_channel = '0'  
second_channel = '1'
third_channel = '2'

selected_channels = []
for i, c in enumerate([first_channel, second_channel, third_channel]):
    if c == 'None':
        continue
    if int(c) >= img.shape[-1]:
        assert False, 'Invalid channel'
    selected_channels.append(int(c))

img_selected = np.zeros_like(img)
img_selected[:, :, :len(selected_channels)] = img[:, :, selected_channels]

masks, flows, styles = model.eval(
    img_selected,
    batch_size=32,        
    flow_threshold=0.4,  
    cellprob_threshold=0.0,
    normalize={"tile_norm_blocksize": 0},invert=True
)

fig = plt.figure(figsize=(14, 6))
plot.show_segmentation(fig, img_selected, masks, flows[0])
plt.tight_layout()
plt.show()

**Comment by yuu342** [comment]
<img width="4170" height="1101" alt="Image" src="https://github.com/user-attachments/assets/3d257dac-3691-44e4-81ed-09f1e719d5e6" />

**Comment by mrariden** [comment]
Hi @yuu342 for programming help please use the [image.sc](http://image.sc) forum. The issues here are for installation errors and bug reports
